src/Homework 5/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace ATM_Managment
+{
+    public class ATM
+    {
+        public delegate void Action(decimal balance, string action);
+        public event Action BalanceHandler;
+
+        private decimal _balance = 0;
+
+        public void AddFunds(decimal _count)
+        {
+            _balance += _count;
+            BalanceHandler?.Invoke(_balance, "deposite");
+        }
+        public void WithdrawFunds(decimal _count)
+        {
+            if (_count > _balance)
+            {
+                Console.WriteLine("You do not have enough funds to withdraw");
+            }
+            else
+            {
+                _balance -= _count;
+                BalanceHandler?.Invoke(_balance, "withdrawal");
+            }
+        }
+        public void ShowBalance()
+        {
+            BalanceHandler?.Invoke(_balance, "\"Show balance\"");
```

Comment:
"\"Show balance\"" means that you are passing "Show balance" message instead of Show balance (without double quotes) as with other messages. Is it a specific case?

Reply:
Yes, cause I think "Show balance" - it is the name of transaction
And deposit transaction - it sounds good like it is, without quotes